pkg/khealth/khealth.go

```diff
@@ -0,0 +1,70 @@
+package khealth
+
```

Comment:
Perhaps just a small description at the beginning on what's the purpose of the package.